services/skus/receipt_test.go

```diff
@@ -543,10 +789,16 @@ func TestFindInAppBySubID(t *testing.T) {
 			name: "one_item_not_found",
 			given: tcGiven{
 				iap: []appstore.InApp{
-					{ProductID: "bravevpn.yearly"},
+					{
```

Comment:
suggestion(non-blocking): we could rename the test to describe why the items is not found` e.g. one_item_not_found_product_id_sub_id_mismatch`

When looking at the test its not immediately obvious why its not found due to all items being expired or product id not equal to the sub id.

Reply:
Thanks, updated.